src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR_COST_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const MIDDLE_TERM = 6;
+  const SHORT_TERM_DISCOUNT = 20;
+  let totalCost = RENT_CAR_COST_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    totalCost -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM && days <= MIDDLE_TERM) {
+    totalCost -= SHORT_TERM_DISCOUNT;
+  }
+
+  return totalCost;
```

Comment:
you can get fix ai comment and make your code cleaner by adding the return right from the if statement:

```suggestion
  if (days >= LONG_TERM) {
    return totalCost - LONG_TERM_DISCOUNT;
  } 
  
  else if (days >= SHORT_TERM) {
    return totalCost - SHORT_TERM_DISCOUNT;
  }
  
  return totalCost;
```

Thus you can make `totalConst` constant

Reply:
As a bonus - you don't really need MIDDLE_TERM with such fix